relentless/simulate/hoomd.py

```diff
@@ -666,56 +696,61 @@ def __call__(self, timestep):
                 self._V[key] += val
 
         def reset(self):
-            """Reset sampling variables to 0."""
+            """Resets sample number, ``T``, ``P``, and all ``V`` parameters to 0."""
             self.num_samples = 0
             self._T = 0.
             self._P = 0.
-            self._V = {'Lx' : 0., 'Ly': 0., 'Lz': 0., 'xy': 0., 'xz': 0., 'yz': 0.}
+            if not hasattr(self.logger, "Lz"):
+                self._V = {'Lx' : 0., 'Ly': 0., 'xy': 0.}
+            elif hasattr(self.logger, "Lz"):
+                self._V = {'Lx' : 0., 'Ly': 0., 'Lz': 0., 'xy': 0., 'xz': 0., 'yz': 0.}
```

Comment:
Based on other order of code, flip these to put the 3d case first. This should also be `if-else` not `if-elif` because this is boolean (either it `hasattr` or it doesn't).